### PR TITLE
docs: added docs for openshift and kubernetes

### PR DESCRIPTION
**Pull request by prahaladhchandrahasan**
Migraing docs from [hawtio-online](https://github.com/hawtio/hawtio-online) to the wbesite .Solves the issue https://github.com/hawtio/website/issues/14

### CODE REVIEW

File: src/templates/docs-template.js [hunk 1]
```diff
@@ -44,6 +44,18 @@ const DocsTemplate = ({ data }) => {
                 Keycloak
               </NavItem>
             </Nav>
+            <h5>Hawtio in Containers</h5>
+            <Nav stacked>
+            <NavItem href={withPrefix('/docs/create-user/')}>
+                Creating Users
+              </NavItem>
+              <NavItem href={withPrefix('/docs/generate-certificates/')}>
+              Generating Certificates
```
Comment (tadayosi) [inline src/templates/docs-template.js]: Please indent correctly

File: src/templates/docs-template.js [hunk 2]
```diff
@@ -44,6 +44,18 @@ const DocsTemplate = ({ data }) => {
                 Keycloak
               </NavItem>
             </Nav>
+            <h5>Hawtio in Containers</h5>
```
Comment (tadayosi) [inline src/templates/docs-template.js]: We should simply write `Kubernetes/OpenShift` here. "Containers" is not the best word for hawtio-online.

**Review by tadayosi — CHANGES_REQUESTED** [review]
Thanks!

Can we move all the added `*.md` files into a separate dir `src/docs/online`?

Also where is the content of hawtio-online/README.md?

File: src/templates/docs-template.js (hunk 1 above)
Comment (prahaladhchandrahasan) [reply src/templates/docs-template.js]: yes sure

File: src/templates/docs-template.js (hunk 2 above)
Comment (prahaladhchandrahasan) [reply src/templates/docs-template.js]: okay

File: src/docs/online/openshift.md [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+## Authentication
+
+Hawtio Online currently supports two authentication modes: `oauth`, which is configured through `HAWTIO_ONLINE_AUTH` environment variable on Deployment.
+
+| Mode | Description |
+| ---- | ----------- |
+| oauth | Authenticates requests through [OpenShift OAuth server](https://docs.openshift.com/container-platform/4.9/authentication/understanding-authentication.html). It is available only on OpenShift. |
+
+
+## Development
```
Comment (tadayosi) [inline src/docs/online/openshift.md]: The Development section can be extracted from the OpenShift and Kubernetes docs as the dedicated doc `develop.md`.

**Review by tadayosi — CHANGES_REQUESTED** [review]
Thanks. Only one more change request.

File: src/docs/online/openshift.md (hunk 3 above)
Comment (prahaladhchandrahasan) [reply src/docs/online/openshift.md]: okay :+1:

**Review by tadayosi — APPROVED** [review]
Looks great!

**Pull request merged**